functions/diff.js

```diff
@@ -17,6 +17,19 @@ const diff = (version1, version2) => {
         }
       }
     }
+    if (v1.major === v2.major &&
+        v1.minor === v2.minor &&
+        v1.patch === v2.patch && !(v1.prerelease.length && v2.prerelease.length)) {
```

Comment:
Based on line 15 and line 12, we can simplify this to:

```suggestion
if (!hasPre) {
```